src/main/java/com/vadmack/mongodbtest/exception/handler/CustomExceptionHandler.java

```diff
@@ -0,0 +1,16 @@
+package com.vadmack.mongodbtest.exception.handler;
+
+import com.vadmack.mongodbtest.exception.NotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class CustomExceptionHandler {
```

Comment:
This class is not required for now. You can use the following approach:
https://www.baeldung.com/exception-handling-for-rest-with-spring#3-responsestatusexceptionresolver

Reply:
"The same as the DefaultHandlerExceptionResolver, this resolver is limited in the way it deals with the body of the response — it does map the Status Code on the response, but the body is still null."

In the issue #3 it said "In this case server should return 404 response code with message: "Project with id='111' not found""